language-server/modules/langserver-core/src/test/resources/definition/sources/projectls/modules/lsmod3/lsmod3.bal

```diff
@@ -1,4 +1,7 @@
 import ballerina/module1;
+import projectls.lsmod1;
 
 public function mod3Function1() {
 }
+
+public type MyType lsmod1:MyType;
```

Comment:
Lets add one test scenario like this

```
public type MyType2 MyType;
```

and add 

```
lsmod3:MyType 2_ = 123;
```

inside the `projectls/typereference`.

this is to check nested types on the safe file/module.
wdyt?